README.md

```diff
@@ -16,12 +16,19 @@ Routing and networking for Cloud Foundry running on Kubernetes.
 - `helm` installed
 
 ### Istio
-* Install [Istio](https://istio.io/docs/setup/install/kubernetes/) to the Kubernetes cluster
-
-    **Note:** As an example, in our CI we are installing Istio via the [deploy-istio.sh](ci/tasks/istio/deploy-istio.sh) task.
+* Install [Istio](https://istio.io/docs/setup/install/kubernetes/) on the Kubernetes cluster
+    
+    * We are testing with the "default" configuration using [these config overrides](ci/istio-config/overrides.yaml).
```

Comment:
Is this clear enough? Trying to figure out how best to convey the settings we're using.